### PR TITLE
MOR-62: Show Latest Review Comment In "Check Your Answers"

### PR DESCRIPTION
**Pull request by lancewalton**

### CODE REVIEW

File: app/uk/gov/hmrc/gform/submission/DestinationAuditAlgebra.scala [hunk 1]
```diff
@@ -18,25 +18,33 @@ package uk.gov.hmrc.gform.submission
...
 import uk.gov.hmrc.gform.sharedmodel.form._
-import uk.gov.hmrc.gform.sharedmodel.formtemplate.{ FormComponentId, FormTemplateId }
 import uk.gov.hmrc.gform.sharedmodel.formtemplate.destinations.{ Destination, DestinationId }
+import uk.gov.hmrc.gform.sharedmodel.formtemplate.{ FormComponentId, FormTemplateId }
 import uk.gov.hmrc.http.HeaderCarrier
-import uk.gov.hmrc.mongo.json.ReactiveMongoFormats.localDateTimeWrite
+import uk.gov.hmrc.mongo.json.ReactiveMongoFormats._
 
-import scala.concurrent.ExecutionContext
+import scala.concurrent.ExecutionContext // Combinator syntax
```
Comment (josef-vlach) [inline app/uk/gov/hmrc/gform/submission/DestinationAuditAlgebra.scala]: `// Combinator syntax` ?

File: app/uk/gov/hmrc/gform/submission/DestinationAuditAlgebra.scala [hunk 2]
```diff
@@ -18,25 +18,33 @@ package uk.gov.hmrc.gform.submission
...
 
-case class SummaryHtmlId(value: UUID) extends AnyVal
+case class SummaryHtmlId(value: UUID)
```
Comment (josef-vlach) [inline app/uk/gov/hmrc/gform/submission/DestinationAuditAlgebra.scala]: Cannot we keep `extends AnyVal`?

File: app/uk/gov/hmrc/gform/submission/DestinationAuditAlgebra.scala (hunk 1 above)
Comment (lancewalton) [reply app/uk/gov/hmrc/gform/submission/DestinationAuditAlgebra.scala]: Weird. Not sure what happened there. Removed.

File: app/uk/gov/hmrc/gform/submission/DestinationAuditAlgebra.scala (hunk 2 above)
Comment (lancewalton) [reply app/uk/gov/hmrc/gform/submission/DestinationAuditAlgebra.scala]: Yes. Overenthusiastic reversion, I think. Replaced.

**Pull request closed without merging**